### PR TITLE
Refactor Transactions `Gov`

### PR DESCRIPTION
**Pull request by austinchandra**
Refactor Transactions `Gov` using #109.

Note that `MsgSubmitProposal` is untested with EIP-712 and likely will not work. This fix is out-of-scope for this PR, and we will address it separately.

### CODE REVIEW

File: docs/transactions/gov/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Gov
```
Comment (GAtom22) [inline docs/transactions/gov/README.md:1]: Nit
```suggestion
# Governance
```

File: docs/transactions/gov/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+This package creates transaction payloads with messages from the [Gov Module](https://docs.cosmos.network/v0.47/modules/gov).
```
Comment (GAtom22) [inline docs/transactions/gov/README.md:3]: Nit
```suggestion
This package creates transaction payloads with messages from the [Governance Module](https://docs.cosmos.network/v0.47/modules/gov).
```

**Review by GAtom22 — APPROVED** [review]
LGTM! Great work @austinchandra ! Left a couple of nits

File: docs/transactions/gov/README.md (hunk 2 above)
Comment (austinchandra) [reply docs/transactions/gov/README.md:3]: Kept as is to maintain parity with the Cosmos SDK's naming (which is Gov).

**Pull request merged**